tasks/bundle_analysis_notify.py

```diff
@@ -64,6 +63,15 @@ def run_impl(
         except LockRetry as retry:
             self.retry(max_retries=5, countdown=retry.countdown)
 
+    def get_success_value(
+        self, configured_notifications_count: int, successful_notifications_count: int
+    ) -> str:
+        if configured_notifications_count == 0:
+            return "nothing_to_notify"
+        if configured_notifications_count == successful_notifications_count:
+            return "full_success"
+        return "partial_success"
```

Comment:
would it make sense to create a strictly typed, instead of "stringly typed", enum for this?

Reply:
It would look cleaner, but these are mostly for debug so it doesn't matter as much.
But ok I can create a proper enum 👍